pom.xml

```diff
@@ -76,14 +76,14 @@
 
   <properties>
     <awaitility.version>3.1.6</awaitility.version>
-    <cdap.version>6.8.0-SNAPSHOT</cdap.version>
+    <cdap.version>6.11.0-SNAPSHOT</cdap.version>
     <commons.version>3.9</commons.version>
     <common.codec.version>1.12</common.codec.version>
     <gson.version>2.8.5</gson.version>
     <googleauth.version>0.4.0</googleauth.version>
-    <hadoop.version>2.3.0</hadoop.version>
+    <hadoop.version>3.3.6</hadoop.version>
     <httpcomponents.version>4.5.9</httpcomponents.version>
-    <hydrator.version>2.10.0-SNAPSHOT</hydrator.version>
+    <hydrator.version>2.10.0</hydrator.version>
```

Comment:
bump it to `2.13.0-SNAPSHOT` for consistency

Reply:
Bumped 👍